### ISSUE TITLE
ROSEE GUI improvements

### ISSUE THREAD

**Issue by lia2790** [ADVRHumanoids/ROSEndEffectorDocs#18]
I open this issue with the aim to suggest two improvements about the ROSEE GUI

**Comment by lia2790** [comment]
One it is relative to the name of the ROSEE GUI gui_main"...  I do not like it. and the other one is the logo, there is not, could we put it?

To see 


![Screenshot from 2020-07-04 12-00-19](https://user-images.githubusercontent.com/15608027/86510297-d284b500-bdee-11ea-89a2-880709f0dec0.png)

**Comment by lia2790** [comment]
I occured, in the GUI for the Schunk hand, that the name size is not fitted with the size of the window as you can see from the picture below

![Screenshot from 2020-07-04 14-38-01](https://user-images.githubusercontent.com/15608027/86512676-76c52680-be04-11ea-859c-60b22eb8ca0c.png)

**Comment by torydebra** [comment]
Yes for name, I can't understand what you mean for logo. Is the icon in the linux dash?
About layout, I have improved a bit in the devel2 branch. Anyway, considering different screen size is not so easy :D, I pointed more in making the gui usable.

**Comment by lia2790** [comment]
> Yes for name, I can't understand what you mean for logo. Is the icon in the linux dash?
yes it is. you see, there is the question marker and I though if it is possible to modify it, it could be better (sorry I did not know how to explained it )
> About layout, I have improved a bit in the devel2 branch. Anyway, considering different screen size is not so easy :D, I pointed more in making the gui usable.

yeah of course, I just report what I think it could be improved :)

**Comment by torydebra** [comment]
About the icon 
`a.setWindowIcon(QIcon("./images/icon.png"));`
obviosly we need an icon :D
